screens/SettingsPage.jsx

```diff
@@ -0,0 +1,57 @@
+import { StyleSheet, Text, View, TouchableOpacity } from 'react-native'
+import React from 'react'
+import { Button } from "@react-native-material/core";
+import { auth } from "../config/firebase";
+import { signOut } from "firebase/auth";
+import { useContext } from 'react';
+import { UserContext } from '../contexts/userContext'
+import { patchUserPreferences } from "../utils/api.js";
+
+const SettingsPage = () => {
+
+    const {user} = useContext(UserContext)
+
+    const handleSignOut = () => {
+        signOut(auth).then(() => {
+          console.log('Signed out')
+        }).catch((error) => {
+          console.log(error)
+        });
+      }
+
+    const resetRecommendations = async () => {
+        try {
+            const data = 'cheese'
+            console.log(user);
+            await patchUserPreferences(user, {preferences: data});
+
```

Comment:
so on reset, it will show the user the 10 hardcoded items from the backend

so should more effort be taken to show the user 10 items with a wide variety of tags in lieu of a tutorial for now